src/tech/v3/libs/smile/discrete_nb.clj

```diff
@@ -15,10 +15,10 @@
      freq-map)
     sparse-array))
 
-(defn bow->SparseArray [ds bow-col indices-col vocab-size]
+(defn bow->SparseArray [ds bow-col indices-col create-vocab-fn]
```

Comment:
I changed as well the configuration of the vocabulary to be used.
Passing a function is more flexible then the 'vocab-size" parameter we had so far